### ISSUE TITLE
Renaming a notebook creates a copy

### ISSUE THREAD

**Issue by mardukbp** [andrewray/iocaml#34]
If I change the name of the notebook I am currently working on, a copy of this notebook with the new name is created.

**Comment by andrewray** [comment]
Yes, that's kind of the way I left it, even though it does not match the original ipython behaviour.  The issue was around what to do when there are 2 web pages accessing the same notebook (and kernel) and then one of them renames and deletes the notebook.  It's not clear what exactly to do and leaving the old one untouched seemed safer.

That said if the 2nd webpage simply stops functioning anyway (which is quite probable) it would be ok to delete it.  I'll have a further think about this.
